src/content/docs/en/guides/cms/storyblok.mdx

```diff
@@ -443,7 +443,9 @@ const story = data.story;
 </html>
 ```
 
-This file will generate a page for each story, with the slug and content fetched from the Storyblok API. 
+This file will generate a page for each story, with the slug and content fetched from the Storyblok API.
+
+> When adding folders inside of storyblok, you'll want to include them when interacting with the Storyblok API. For example, in the get request above we could use **cdn/stories/blog**, with a blog folder inside rather than using them at the root.
```

Comment:
Thanks for this addition, @thejackshelton! In order to figure out how to best include this content, I do have some questions though. You wouldn't think one single paragraph sparks so much discussion, and yet, here we are!  That's my job!  :smile:

- @ElianCodes is right in that we don't use "block quote" syntax for notes. (We use it for.... quotations!) We have a specific note format for asides.
- BUT, we only use notes for truly tangential, "beside the point" information. And I'm not sure whether that's the content you're giving here.

In general the wording is a little unclear to me:
- The "when...., you will want to..." phrasing is a bit tricky and usually ends up with more words than necessary. We'd often go for a more direct "For" instead, e.g. "For content that has been edited.... do this" In fact, we try to avoid "You" at all, and just state the instructions / active verb directly, and under which circumstances when that's not universal advice.

- "could" is a very scary word in docs! "You *could* do this..." Well,... should I? Must I? This is typically unclear advice to the reader, and we always look closely when we see this word in the hopes of making it more explicit. This is telling me to "idk, maybe?" do something, and we try to avoid using the word "could" unless we are truly describing an uncertainty. (It "could" rain...) To let the reader know that they have a choice, we would typical tell them that they "can" do X or Y. This sounds more confident, reassuring, and definitive! 

(And, we are often strategic about when we let them know they have a choice, in hopes of keeping them on a happy path when following the instructions. We don't HAVE to tell them of all the things possible.)

So, we need to establish how key is this information to include so that a reader's project will work when they follow the steps?

- If key, then we do NOT use a note, and we would try to start a sentence with "For this particular situation.... ,(do this)" if that fits. 
- If a kind of "heads up!" aside, less common situation but they really should know, then we would use a note.

I would also prefer to avoid phrases like "you will want to" and "could"?  Would you be willing to take another shot at writing this paragraph, with more direct language that avoids addressing the reader and replaces "could" with more direct advice for the user?